routes/main_routes.js

```diff
@@ -129,7 +132,21 @@ module.exports = function (app, passport) {
             res.redirect('/');
         });
 
-    app.use('/:language/tickets/', ticket_routes);
+    // =====================================
+    // JWT =================================
+    // =====================================
+    // JWT Login route
+    app.post("/jwt-login", function (req, res) {
+        passportLib.login(req.body.email, req.body.password, function(result, user, error) {
+            if (result && user) {
+                var token = passportLib.generateJwtToken(req.body.email);
```

Comment:
jwt-login is there for AJAX login and as a reference implementation. Login via the UI is at '/login' route. I agree that there is a need to enhance this behavior as you suggested.